lib/http/index.js

```diff
@@ -179,13 +179,19 @@ module.exports = function(root) {
         app.use(settings.strategy.favicon(staticPath + '/favicon.ico'))
       }
       if (settings.strategy.session) {
+        var cookie = {
+          path: '/',
+          httpOnly: false,
+          maxAge: sessionOptions.maxAge,
+          secure: settings.secure
+        };
+        if (settings.strategy.sessionCookie) {
+          for(var k in settings.strategy.sessionCookie) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.